securedrop/tests/test_unit_source.py

```diff
@@ -129,6 +129,28 @@ def test_login_and_logout(self):
             self.assertIn('Sorry, that is not a recognized codename.', rv.data)
             self.assertNotIn('logged_in', session)
 
+    def test_login_with_whitespace(self):
+        rv = self.client.get('/login')
+        self.assertEqual(rv.status_code, 200)
+        self.assertIn("Login to check for responses", rv.data)
+
+        codename = self._new_codename()
+	codename = ' ' + codename + ' '
```

Comment:
It would also be good to test likely variants, such as `' ' + codename` and `codename + ' '`.